### PR TITLE
Update to Julia v1.0

### PR DESCRIPTION
**Pull request by wildart**

### CODE REVIEW

File: src/kpca.jl [hunk 1]
```diff
@@ -1,23 +1,25 @@
 # Kernel Principal Component Analysis
 
+import Arpack
```
Comment (ararslan) [inline src/kpca.jl]: Arpack will need to be included in the REQUIRE file (it isn't a stdlib package)

File: .travis.yml [hunk 2]
```diff
@@ -3,12 +3,8 @@ os:
     - osx
     - linux
 julia:
-    - 0.6
+    - 1.0
```
Comment (ararslan) [inline .travis.yml]: Should also test 0.7 since we allow it in REQUIRE

File: .travis.yml [hunk 3]
```diff
@@ -3,12 +3,8 @@ os:
...
     - nightly
 notifications:
     email: false
-# script:
-#     - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-#     - julia -e 'Pkg.clone(pwd()); Pkg.build("MultivariateStats"); Pkg.test("MultivariateStats"; coverage=true)';
-after_success:
-    - julia -e 'cd(Pkg.dir("MultivariateStats")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())';
```
Comment (ararslan) [inline .travis.yml]: We can still do coverage submission, it just needs to be slightly restructured:
```yaml
after_success:
  - julia -e 'using Pkg; Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())'
```

**Comment by ararslan** [comment]
Looks like we'll need a `using SparseArrays` somewhere

**Comment by wildart** [comment]
Finally got it right.

**Comment by ararslan** [comment]
Nice. LGTM then.

**Pull request merged**